lib/resque/scheduler/delaying_extensions.rb

```diff
@@ -257,55 +248,49 @@ def get_last_enqueued_at(job_name)
 
       private
 
+      def decode_without_nonce(job)
+        decode(job).delete_if {|k, _| k == 'nonce'}
```

Comment:
nitpick: `reject` is more appropriate but i assume `decode` is returning a brand new hash anyway?